src/Components/Publishing/Series/ArticleCard.tsx

```diff
@@ -62,10 +63,14 @@ export class ArticleCard extends Component<Props, null> {
   }
 
   render () {
-    const { article, color, series } = this.props
+    const { article, children, color, series } = this.props
     const { media } = article
     const isUnpublishedMedia = this.isUnpublishedMedia()
 
+    const editTitle = children && children[0]
+    const editDescription = children && children[1]
+    const editImage = children && children[2]
```

Comment:
Curious if you had further thoughts about this pattern. This works, but I can see how having to know the order of the children could become more of a burden as others try to build an editable  `ArticleCard`.

What if these components were passed down as optional props instead of looking at the children? Then, this parent component can decide to render it or not based on those props. That way we can be more compliant with TS and have autocomplete instead of having to go to this specific line of code to implement the component.

Reply:
Sure will try this way instead